src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java

```diff
@@ -233,6 +215,60 @@ public OrganisationsDetailResponse retrieveOrganisations() {
         return updatedOrganisationDetails;
     }
 
+    @Override
+    public OrganisationsDetailResponse retrieveOrganisations(String group) {
```

Comment:
Rename this method to retrieveAllOrganisation since this is dedicated method for getting all orgs.
Don't pass any params and remove switch which is not required here.